lerna.json

```diff
@@ -6,6 +6,26 @@
     "access": "public",
     "registry": "https://registry.npmjs.org/"
   },
+  "changelog": {
+    "repo": "patternfly/patternfly-react",
+    "cacheDir": ".changelog",
+    "nextVersion": "Unreleased",
+    "labels": {
+      "PR: Spec Compliancy :eyeglasses:": ":eyeglasses: Spec Compliancy",
```

Comment:
I have used the same labels as Babel's repo. We'll need to adjust these based on what we would like to use in PRs here. I tend to like the icons in the changelogs since they are easy to find the different types of commits.